src/components/EventCard/EventCard.jsx

```diff
@@ -1,15 +1,48 @@
 import React from 'react';
 
 import { S } from './EventCard';
+import { useState } from 'react';
```

Comment:
```suggestion
import React, {useState} from 'react';

import { S } from './EventCard';
```
같은 라이브러리를 사용하는 부분은 이렇게 적용해주세요